cdap-data-fabric/src/main/java/io/cdap/cdap/store/StoreDefinition.java

```diff
@@ -370,6 +370,10 @@ public static final class AppMetadataStore {
     public static final String APPLICATION_FIELD = "application";
     public static final String VERSION_FIELD = "version";
     public static final String APPLICATION_DATA_FIELD = "application_data";
+    public static final String CREATION_TIME_FIELD = "created";
```

Comment:
I think we should add index on `created` and `latest`? @sanjanasandeep

Reply:
yes, was thinking of adding it in the next PR (which updates the endpoints), since it was breaking some tests - and needs changing those tests as well, and I don't want to make the his PR too long